specs/powerfulfeatures/index.src.html

```diff
@@ -191,12 +191,12 @@ <h4 id="threat-active">Active Network Attacker</h4>
   privacy who are able to target individual users, organizations or even
   entire populations.
 
-  <h3 id="threat-risks">Risks associated with non-privileged contexts</h3>
+  <h3 id="threat-risks">Risks associated with non-secure contexts</h3>
```

Comment:
Did you intentionally choose "non-secure" rather than "insecure"? I noted the same phrasing in https://blog.mozilla.org/security/2015/04/30/deprecating-non-secure-http/ and it sounds strange.